Realm/RLMArray.mm

```diff
@@ -299,6 +299,13 @@ - (RLMResults *)objectsWhere:(NSString *)predicateFormat args:(va_list)args
     return [self objectsWithPredicate:[NSPredicate predicateWithFormat:predicateFormat arguments:args]];
 }
 
+- (id)valueForKeyPath:(NSString *)keyPath {
+    if (_backingArray) {
+        return [_backingArray valueForKeyPath:keyPath];
```

Comment:
How could users be depending on this? Support for KVC operators is new in this change.